azkaban-web-server/build.gradle

```diff
@@ -74,6 +78,27 @@ bower {
 }
 clean.dependsOn 'bowerClean'
 
+task jsTest(dependsOn: ['npm_install'], type:Exec) {
+  workingDir '.'
+  ignoreExitValue = true
+
+  commandLine 'node', 'node_modules/mocha/bin/mocha', 'src/web/js/azkaban/test/'
+
+  standardOutput = new ByteArrayOutputStream()
+  doLast {
+    println standardOutput
+    if (execResult) {
+      if(execResult.exitValue == 0) {
+        println "All js Tests passed"
+      }
+      else{
+        throw new GradleException('Js Tests failed.')
+      }
+    }
+  }
+}
+test.dependsOn 'jsTest'
```

Comment:
nit: You can use jsTest instead of 'jsTest'

The quotes make it a string which results in late evaluation. This is useful when referring to external tasks / other cases where late resolution is required. The first preference however is generally to resolve it in configuration phase.